lib/zlib.js

```diff
@@ -129,8 +134,8 @@ function zlibBufferSync(engine, buffer) {
   if (typeof buffer === 'string')
     buffer = Buffer.from(buffer);
   else if (!ArrayBuffer.isView(buffer))
-    throw new TypeError('"buffer" argument must be a string, Buffer, ' +
-                        'TypedArray, or DataView');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'buffer',
+      ['string', 'Buffer', 'TypedArray', 'DataView']);
```

Comment:
please align the second row here with the `ERR_INVALID_ARG_TYPE` in the previous.